### ISSUE TITLE
aria-current is a global aria state. 

### ISSUE THREAD

**Issue by dstorey** [validator/validator#131]
Error:

Error: Attribute aria-current not allowed on element li at this point.
From line 3, column 4752; to line 3, column 4797

```
a> </li>  <li aria-current="page" class="link--current"> <a hr
```

Attributes for element li:
Global attributes

aria-current is a global state in Aria 1.1 and should probably be allowed, or switched to a warning to say that it will be allowed when Aria 1.1 is more mature?

**Comment by sideshowbarker** [comment]
If `aria-current` is being used by authors I assume it's because it must have some level of support in browsers and AT already (otherwise it seems pointless to be using it) but maybe not in all (or most) browsers/AT, in which case it would be appropriate to emit a warning about it.

But if it's in fact that case that no UAs/AT support it yet, it seems appropriate to just emit an error for it, because the HTML spec (or the HTML+ARIA extension spec that @stevefaulkner is the author of) is what defines what ARIA attributes are valid in HTML docs, and `aria-current` is not yet in that.

But if there is some level of UA/AT support for it, then I guess it's time that Steve needs to look into adding it to HTML (along with any other ARIA 1.1 attributes that may be implemented now).

**Comment by stevefaulkner** [comment]
I believe that currently (no pun) aria-current is not supported, but will investigate, and yes I need to add new 1.1 roles/attributes to the ARIA in HTML spec, but need to ascertain their maturity/stability status before doing so.

**Comment by stevefaulkner** [comment]
Have pinged PF to get ball rolling https://lists.w3.org/Archives/Public/public-pfwg/2015Jul/0077.html (not that their input will block addition to ARIA in HTML).

**Comment by stevefaulkner** [comment]
Also opened bug on ARIA in HTML spec https://github.com/webspecs/html-aria/issues/23

**Comment by sideshowbarker** [comment]
Just noting here that `aria-current` is still not yet in https://www.w3.org/TR/html-aria/ and I don’t plan to implement anything for it unless/until it gets added there.

**Comment by sideshowbarker** [comment]
Fixed by ead41c2
